tests/testthat/test-qenv_eval_code.R

```diff
@@ -88,41 +88,85 @@ testthat::test_that("an error when calling eval_code returns a qenv.error object
     unname(q$trace),
     c("x <- 1", "y <- 2", "z <- w * x")
   )
-  testthat::expect_equal(q$message, "object 'w' not found \n when evaluating qenv code:\nz <- w * x")
+  testthat::expect_equal(q$message, "object 'w' not found \n when evaluating qenv code:\nexpression(z <- w * x)")
```

Comment:
This is failining and now contains a bit more.
It is becuase in `eval_code` when we calculate the message error we use
`current_call` that has `keep.source = TRUE` in the `parse`.

Maybe we can substitute `current_call` with `current_code`

```r
x[1]: "object 'w' not found \n when evaluating qenv code:\nstructure(expression(z 
x[1]: <- w * x), srcfile = <environment>, wholeSrcref = structure(c(1L, 0L, 2L, 0L
x[1]: , 0L, 0L, 1L, 2L), srcfile = <environment>, class = \"srcref\"))"
y[1]: "object 'w' not found \n when evaluating qenv code:\nexpression(z <- w * x)"
```

Reply:
Fixed in here 20b2c8fc00128c37d46d4eeeed403e72a658a4fc

Reply:
and updated in here b4e119c98bbe1555aec552920a4ed72c3027be3c